lms/static/js/student_profile/views/learner_profile_factory.js

```diff
@@ -45,10 +45,11 @@
                 required: true,
                 editable: 'always',
                 showMessages: false,
-                title: interpolate_text(
-                    gettext('{platform_name} learners can see my:'), {platform_name: options.platform_name}
+                title: HtmlUtils.interpolateHtml(
```

Comment:
This would be `StringUtils.interpolate()` according to current thinking of best practices.  We do need to determine.